### DISCUSSION TITLE
22. October 2024

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#2043] [Meeting notes]
Agenda:
- [x] Remove @dimakuv from [the Maintainers list](https://github.com/gramineproject/gramine/blob/master/Documentation/management-team.rst)
   - Voting on removing @dimakuv from the Maintainers list, effective starting from 31. October
   - Yes from @donporter 
   - Yes from @monavij 
   - Yes from @woju 
   - Yes from @chiache 
   - Yes from @kailun-qin 
   - Majority of votes was reached (5 out of 8, excluding @dimakuv himself)
- [ ] Also remove @boryspoplawski from it?
   - Decided to wait with this voting until later.

**Comment by woju** [comment]
- [x] 1.8 release announcement (@woju, 5 min)
